### DISCUSSION TITLE
LBB Streamlit : Error ketika deploy streamlit

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#891] [Q&A] [answered]
Izin bertanya Team Algoritma ;

Ketika proses Deploy tdk error sbb :
ImportError: 'convert_anything_to_df' from 'streamlit.type_util' 

![image](https://github.com/user-attachments/assets/ac5e2fc9-e95a-488e-aa62-ebc0c72d2191)

Mohon solusinya...
Terimakasih

**Comment by shinddayo** [comment]
same here

apakah ada yang salah dengan Gsheets nya?
apa yang harus dilakukan kah?
untuk spreadsheet nya sudah di set access anyone with the link

![image](https://github.com/user-attachments/assets/4c18c86d-d4da-4b61-81c6-b5d2dd7926ff)

**Comment by dwiknrd** [comment]
Dear Bapak [Micoalgo](https://github.com/Micoalgo),

Sehubungan dengan adanya pembaruan dari Streamlit baru-baru ini, versi default aplikasi yang terdeploy saat ini adalah streamlit==1.37.0. Hal ini mungkin menyebabkan error pada beberapa dependensi, terutama pada gsheet_connection. Oleh karena itu, kami menyarankan Bapak untuk memperbarui file requirements.txt dengan menyertakan versi Streamlit yang kompatibel, yaitu streamlit==1.36.0. Apakah bapak sudah memastikan pada requirements github repository versi streamlit sudah sesuai?
​
Demikian yang dapat kami sampaikan, apabila ada pertanyaan lebih lanjut silahkan ajukan dengan menjawab comment ini. Terima kasih.

Best Regards,
Team Algoritma

**Comment by Micoalgo** [comment]
Selamat Pagi Mba Dwi dan Tim Algoritma

Setelah saya ubah streamlitnya menjadi versi streamlit==1.36.0 masih tdp
error sbb :

![image](https://github.com/user-attachments/assets/cbb36c9c-7bc8-4e5d-b52f-4cbb50de2df1)

![image](https://github.com/user-attachments/assets/ddb71af6-b2e8-4c29-9fa7-f65d1b2130c7)

[image: image.png]
mohon solusinya...Terimakasih

**Reply by shinddayo** [reply]
Same here. Tetep gak bisa

**Comment by dwiknrd** [comment]
Hai Pak [Micoalgo](https://github.com/Micoalgo) dan Pak [shinddayo](https://github.com/shinddayo),

Apakah sudah dipastikan untuk me-Reboot applikasi streamlitnya? Karena apabila dilihat dari streamlit, dashboard bapak belum melakukan installasi ulang, hanya saja perubahan versi streamlit.

Regards,
Team Algoritma

**Comment by Micoalgo** [comment]
me reboot = me refresh kah ?
(sebelumnya sdh saya refresh)

**Reply by dwiknrd** [reply]
Hai Pak [Micoalgo](https://github.com/Micoalgo),

Me-reboot aplikasi dengan cara seperti ini pak
<img width="223" alt="image" src="https://github.com/user-attachments/assets/c40cd2ca-73d7-4442-91a4-43d06b1cd9bf">

Apabila sudah dilakukan dan masih belum bisa boleh coba dihapus dan lakukan deploy ulang. Demikian yang dapat kami sarankan pak. Semoga membantu.

Best Regards,
Team Algoritma

**Comment by Micoalgo** [comment]
dan setelah di reboot muncul error sbb :
[image: image.png]

mohon solusinya...Terimakasih

**Reply by dwiknrd** [reply]
Hai Pak [Micoalgo](https://github.com/Micoalgo), 

Terkait screenshot error belum terlampir pak. Mohon untuk mengirimkan ulang. Terima kasih

Best Regards,
Team Algoritma

**Reply by Micoalgo** [reply]
berikut saya coba kirim lagi ya ... mohon solusinya...Terimakasih

![image](https://github.com/user-attachments/assets/62734e90-51fe-4a7e-94c7-49ecc33c17f1)

**Reply by dwiknrd** [reply, marked as answer]
Hai Pak [Micoalgo](https://github.com/Micoalgo),

Terdapat error yang berbeda yaa pak. Hal tersebut dikarenakan bapak belum menyertakan plotly.express dan plotly di file requirements.txt yang bapak deploy.

Semoga membantu, terima kasih.

Best Regards,
Team Algoritma

**Comment by Micoalgo** [comment]
[image: image.png]

**Comment by Micoalgo** [comment]
Ok Mba Dwi...Done...Thank alot ya

**Comment by Micoalgo** [comment]
sdh running app nya setelah saya cek

https://rs-salam-sehat-preview-hfa.streamlit.app/

Terimakasih

**Reply by dwiknrd** [reply]
Dear Pak [Micoalgo](https://github.com/Micoalgo),

Sama-sama pak, senang bisa membantu 😊

Best Regards,
Team Algoritma
